src/platforms/esp32/components/avm_builtins/i2c_driver.c

```diff
@@ -28,37 +28,32 @@
 #include <freertos/FreeRTOS.h>
 #include <freertos/task.h>
 
-#include "atom.h"
-#include "bif.h"
-#include "context.h"
-#include "debug.h"
-#include "defaultatoms.h"
-#include "globalcontext.h"
-#include "interop.h"
-#include "mailbox.h"
-#include "module.h"
-#include "platform_defaultatoms.h"
-#include "scheduler.h"
-#include "term.h"
-#include "utils.h"
-
-//#define ENABLE_TRACE
```

Comment:
Did we take any final decision on the #include style?

Reply:
reverted